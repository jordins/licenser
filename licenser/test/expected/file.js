/*
This is my license file
and ends like this.
*/

console.log("hello");
