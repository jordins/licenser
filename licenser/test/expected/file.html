<!--
This is my license file
and ends like this.
-->

<h1>Hello</h1>
